gr-filter/grc/filter_hilbert_fc.block.yml

```diff
@@ -32,4 +33,14 @@ templates:
     imports: from gnuradio import filter
     make: filter.hilbert_fc(${num_taps}, ${win}, ${beta})
 
+cpp_templates:
+    includes: ['#include <gnuradio/filter/hilbert_fc.h>']
+    declarations: 'filter::hilbert_fc::sptr ${id};'
+    make: |-
+        this->${id} = filter::hilbert_fc::make(
+            ${num_taps},
+            ${win},
+            ${beta});
+    link: ['gnuradio-filter']
+
```

Comment:
We're gonna have to translate `firdes.` to `firdes::` because of the `win` parameter.